react/features/app/actions.js

```diff
@@ -49,6 +47,15 @@ export function appNavigate(uri) {
                     .domain;
         }
 
+        if (room) {
```

Comment:
Yes, splitting and getting last element allows us always get pathname + search + hash